site/docs/foundations/shadow.mdx

```diff
@@ -19,4 +26,48 @@ There are six shadow elevations to choose from.
 | `--salt-shadow-400` | 0           | 6           | 10        |
 | `--salt-shadow-500` | 0           | 12          | 40        |
 
+## Scrollable area: Shadow-100
+
+Shadow-100 is the closest to the surface and is used exclusively for scrollable areas where the content exceeds the visible screen space, allowing users to scroll up or down. This is typically positioned at the top of an overflowing area, providing a subtle hint of depth without casting a noticeable shadow.
+
+## Region: Shadow-200
+
+Shadow-200 introduces a slight elevation, offering a soft shadow that begins to separate elements from the background. This level is suitable for components that require a gentle lift, such as cards or small panels.
+
+<Image
+  src="/img/foundations/shadow-container.png"
+  alt="Diagram showing a region shadow on a container."
+/>
+
+## Hover: Shadow-300
+
+Shadow-300 provides a moderate level of separation with a more pronounced shadow. It is ideal for elements that need to stand out more distinctly, such as dropdown menus or tooltips.
+
+## Pop-out: Shadow-400
+
+Shadow-400 offers a strong contrast with well-defined edges, making it perfect for components that require significant emphasis such as a popout. A popout is a window above the interface with a shadow that temporarily overlays elements, e.g. Toast, Tooltip, Overlay.
+
+<Image
+  src="/img/foundations/shadow-popout.png"
+  alt="Diagram showing a pop-out shadow."
+/>
+
+## Modal: Shadow-500
+
+Shadow-500 is the furthest away from the surface and is reserved for modal dialogs only. It creates a dramatic effect with a strong shadow, ensuring that modal dialogs are clearly distinguished from the rest of the content. For a modal, the user must interact with the overlay window, which typically has a scrim behind it, before interacting with the page content.
+
+<Image
+  src="/img/foundations/shadow-modal.png"
+  alt="Diagram showing a modal shadow."
+/>
+
+## Scrim
```

Comment:
Personally Scrim feels shoehorned into "Shadows". I guess at the moment there's nowhere else to put it...

Reply:
Spoke with Ben and going to keep